searx/templates/simple/result_templates/packages.html

```diff
@@ -58,6 +58,14 @@
         {%- if result.source_code_url -%}
           <a href="{{ result.source_code_url }}" target="_blank">{{ _('Source code') }}</a>
         {%- endif -%}
+        {%- if result.links %}
+          {%- for name, link in result.links.items() -%}
+            {% if not loop.first or result.homepage or result.source_code_url %} | {% endif %}
+            <a href="{{ link }}" target="_blank">
+              {{- _(name) -}}
```

Comment:
argueable .. but if we have a translation, then lets translate. Common words like `author` are already in our translation memory ..

EN ("Author"):

![image](https://github.com/searxng/searxng/assets/554536/9a986f04-b52a-43b6-9cda-ffe4b54f24bc)


DE ("Autor"):  
![image](https://github.com/searxng/searxng/assets/554536/079b8db6-c317-4a97-9501-96432286f4c9)

Common terms can be added to [searx/searxng.msg](https://github.com/searxng/searxng/blob/master/searx/searxng.msg) where the translation of `author` also comes from (a term that is frequently used in social media):

https://github.com/searxng/searxng/blob/645a840d8211a016d808cf7da1ab62ff12d1db56/searx/searxng.msg#L87-L97